test/test_nexus2clustal.py

```diff
@@ -3,9 +3,10 @@
 import hashlib
 from easydev import TempFile, md5
 
-from bioconvert import bioconvert_data
 from bioconvert.nexus2clustal import NEXUS2CLUSTAL
 
+from . import test_dir
+
 skiptravis = pytest.mark.skipif("TRAVIS_PYTHON_VERSION" in os.environ
```

Comment:
ici on peut aussi enlever the travis decorator car travis est remplacé par le CI action